lib/cavendish/commands/add_eas_configuration.rb

```diff
@@ -3,13 +3,21 @@ module Commands
     class AddEASConfiguration < Cavendish::Commands::Base
       def perform
         install_eas_cli
+        register_and_create_organization_prompt
         login_to_expo
         set_basic_project_config
         inject_advanced_eas_config
       end
 
       private
 
+      REGISTER_MESSAGE = "To configure Expo Application Services(EAS) into your app,"\
+       " it is necessary to have an expo account. Please register at https://expo.dev/signup"\
+       " if you don't have an account (enter to continue)"
+      CREATE_ORGANIZATION_MESSAGE = "If you are planning to collaborate with others in this"\
+       " project, it is adviced to create an organization. Create an organization at https://expo.dev/create-organization"\
+        " (enter to continue)"
```

Comment:
You missed a space between `Expo Application Services` and `(EAS).

Also, I tend to think for these cases that it might be better to use the [heredoc ruby notation](https://www.rubyguides.com/2018/11/ruby-heredoc/):

```ruby
REGISTER_MESSAGE = <<~MSG
  To configure Expo Application Services (EAS) into your app, it is necessary to have an expo account. Please register at https://expo.dev/signup if you don't have an account (enter to continue).
MSG

CREATE_ORGANIZATION_MESSAGE = <<~MSG
  If you are planning to collaborate with others in this project, it is adviced to create an organization. Create an organization at https://expo.dev/create-organization (enter to continue)
MSG
```

Mainly because it improves the syntax (no backslashes or leading / ending spaces on concatenated strings). The main advantage is that Rubocop won't throw an error with heredoc strings. What do you think?

Reply:
I didn't knew heredoc notation :open_mouth: 
Changed both messages to that notation, and fixed the typo